benchmark/index.js

```diff
@@ -236,6 +238,60 @@ test('query three indexes (second run)', (t) => {
   })
 })
 
+test('load two indexes concurrently', (t) => {
+  const alice = ssbKeys.loadOrCreateSync(path.join(dir, 'secret'))
+  const bob = ssbKeys.loadOrCreateSync(path.join(dir, 'secret-b'))
+
+  db.onReady(() => {
+    const done = multicb({ pluck: 1 })
+    const start = Date.now()
+
+    db.indexes['type_about'] = {
+      offset: 0,
+      bitset: new TypedFastBitSet(),
+      lazy: true,
+      filepath: path.join(indexesDir, 'type_about.index'),
+    }
```

Comment:
Is this index going to be loaded by the queries below? It's a lazy index, and we don't seem to be querying `type_about`...

Reply:
Whoops fixed.